subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/test/matcher/BitmapMatcher.kt

```diff
@@ -42,10 +52,10 @@ class BitmapMatcher(private val bitmap: Bitmap) :
         return r < CONFIDENCE_INTERVAL && g < CONFIDENCE_INTERVAL && b < CONFIDENCE_INTERVAL
     }
 
-    private fun Bitmap.getTopLeftPixels(): List<Int> {
+    private fun Bitmap.getPixelsSnapshot(): List<Int> {
         val result = mutableListOf<Int>()
-        for (i in 5..30 step 5) {
-            for (j in 5..30 step 5) {
+        for (i in 1 until this.width step 5) {
```

Comment:
Why do we do a step of 5 pixels? It is too expensive to fairly check all pixels of images? The most of images have 32x32 pixel size. It is only 1024 int value to check. Seems not too much.

Reply:
Didn't want to change previous logic, cause I don't know the motivation for it.
Agree, will compare all the pixels.

Reply:
Fixed